### PR TITLE
Change jQuery toggle() to show/hide to account for DOM parsing lag #5019

### PR DESCRIPTION
**Pull request by cklosowski**
#5019

### CODE REVIEW

File: includes/admin/discounts/add-discount.php [hunk 1]
```diff
@@ -57,8 +57,8 @@
 				</th>
 				<td>
 					<input type="text" id="edd-amount" name="amount" value="" style="width: 40px;"/>
-					<p class="description edd-amount-description" style="display:none;"><?php printf( __( 'Enter the discount amount in %s', 'easy-digital-downloads' ), edd_get_currency() ); ?></p>
-					<p class="description edd-amount-description"><?php _e( 'Enter the discount percentage. 10 = 10%', 'easy-digital-downloads' ); ?></p>
+					<p class="description edd-amount-description flat" style="display:none;"><?php printf( __( 'Enter the discount amount in %s', 'easy-digital-downloads' ), edd_get_currency() ); ?></p>
```
Comment (chriscct7) [inline includes/admin/discounts/add-discount.php]: Just thinking that flat might not be the best idea for a class name given some plugins might globally load css, and use flat for like flat design or something like that. Maybe `discount-type-flat` would be better?

File: assets/js/admin-scripts.js [hunk 2]
```diff
@@ -773,8 +789,10 @@ jQuery(document).ready(function ($) {
 		type_select : function() {
 
 			$('#edd-edit-discount #edd-type, #edd-add-discount #edd-type').change(function() {
-
-				$('.edd-amount-description').toggle();
+				var val = $(this).val();
+				console.log(val);
```
Comment (chriscct7) [inline assets/js/admin-scripts.js]: Should this be removed?

File: includes/admin/discounts/add-discount.php (hunk 1 above)
Comment (cklosowski) [inline includes/admin/discounts/add-discount.php]: @chriscct7 I'll change `flat` to be a bit more defined.

File: assets/js/admin-scripts.js (hunk 2 above)
Comment (cklosowski) [inline assets/js/admin-scripts.js]: Will remove this.

**Pull request merged**